### PR TITLE
RDKDEV-253: WebkitBrowser changes sync up from metro version to rdkservices repo

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

**Review by emutavchi — COMMENTED** [review]
@HaseenaSainul what Thunder version is required that supports loading plugins that are split into 2 libraries? We may need to sync this change with Thunder upgrade.

File: WebKitBrowser/CMakeLists.txt [hunk 1]
```diff
@@ -236,6 +262,11 @@ if(PLUGIN_WEBKITBROWSER_UX)
     write_config( UX )
 endif()
 
+# Resident App configuration
```
[the remark this answers is not in the document]
Comment (HaseenaSainul) [reply WebKitBrowser/CMakeLists.txt:265]: It work with R2 version which is currently available with RDK hybrid or media client build. There is no additional changes required in thunder

[the remark this answers is not in the document]
Comment (HaseenaSainul) [reply WebKitBrowser/CMakeLists.txt]: done

**Review by emutavchi — APPROVED** [review]
Looks good to me. Please cherry-pick to sprint/2203 as well.

File: WebKitBrowser/CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (emutavchi) [reply WebKitBrowser/CMakeLists.txt]: Thanks

**Comment by HaseenaSainul** [comment]
> Looks good to me. Please cherry-pick to sprint/2203 as well.

Sure. I waiting for the merge of one the previous dependent PR from Pradeepta. Will do after that :)

**Pull request closed without merging**